.claude/rules/data-access-boundary.md

```
@@ -0,0 +1,79 @@
+paths:
+  - src/components/**
+  - src/hooks/**
+  - src/app/actions/**
+  - src/app/api/**
+  - src/lib/supabase/**
+---
+# データアクセス境界ルール
+
+## 原則: クライアント → Server Action → DB
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)